### PR TITLE
Add cluster support suspended notification

### PR DESCRIPTION
**Pull request by sulmer-redhat**

### CODE REVIEW

File: osd/cluster_support_suspended.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Cluster support suspended: none-authorised IaaS changes detected",
+    "description": "Openshift Dedicated SRE team has noticed some none-supported changes that were made to the underlying IaaS infrastructure. This cluster will no longer be monitored and managed by the SRE team. Possible causes may include stopping or removing instances, configuration changes to Load Balancers or Security Groups, etc. If you wish to resume support for this cluster, please submit a support ticket.",
```
Comment (cattias) [inline osd/cluster_support_suspended.json:6]: none-supported -> non supported

Comment (cblecker) [reply osd/cluster_support_suspended.json:6]: We also need some input from @wgordon17 on specific terminology here. I've started an e-mail thread to get us all on the same page

**Comment by mmazur** [comment]
The SOP that references this file is already in use. Please finalize the PR and get it merged.

**Comment by sulmer-redhat** [comment]
@wgordon17 waiting on your review so we can merge

**Comment by mmazur** [comment]
I think this can be closed. Whatever messaging comes out OSD-6723 will take this PR's place.

**Comment by wgordon17** [comment]
We can't say anything about loss of support, monitoring or management. This is literally the only thing the customer is paying us for. It would be a breach of contract to say this https://docs.google.com/document/d/1HlKfewZamaK-I0LdRLYN-sh4QLw5SU_0aYaPU6ZD-0g/edit?disco=AAAAIj69HCE

**Comment by mmazur** [comment]
This is being worked on in a different place. As Will mentioned, we can't pull support from a cluster.

**Pull request closed without merging**